src/components/header/header.styles.js

```diff
@@ -2,26 +2,30 @@ import styled from 'styled-components';
 import { NavLink } from 'react-router-dom';
 
 export const NavContainer = styled.nav`
-  height: 100px;
-  margin-left: 80px;
-  margin-right: 80px;
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
+  margin-bottom: 55px;
 `;
 
 export const LogoContainer = styled.div`
   display: block;
+  margin-left: 80px;
+  margin-top: 33px;
 `;
 
 export const Links = styled.div`
   font-family: 'Montserrat';
```

Comment:
You can define the font-family globally on the `body` element. Then you don't need to set it here again.